assets/css/style.css

```diff
@@ -4,6 +4,24 @@
   height: 100%;
 }
 
+/*Social Media Icons*/
+/* Style all font awesome icons */
+.fa {
```

Comment:
What are these CSS definitions for? I don't think we need them, do we?

Reply:
They're being used for the social media icons on the navbar.

![image](https://user-images.githubusercontent.com/54612505/121934229-a98dd380-cd64-11eb-9904-e942a08aa961.png)